src/server/chromium/crNetworkManager.ts

```diff
@@ -288,15 +288,19 @@ export class CRNetworkManager {
     const timingPayload = responsePayload.timing!;
     let timing: network.ResourceTiming;
     if (timingPayload) {
+      // requestTime is a baseline in seconds, while other timings are ticks in milliseconds relatively to this requestTime.
+      const calculateOffset = (value: number) => {
+        return timingPayload.requestTime + (value / 1000);
+      };
       timing = {
-        startTime: (timingPayload.requestTime - request._timestamp + request._wallTime) * 1000,
-        domainLookupStart: timingPayload.dnsStart,
-        domainLookupEnd: timingPayload.dnsEnd,
-        connectStart: timingPayload.connectStart,
-        secureConnectionStart: timingPayload.sslStart,
-        connectEnd: timingPayload.connectEnd,
-        requestStart: timingPayload.sendStart,
-        responseStart: timingPayload.receiveHeadersEnd,
+        startTime: timingPayload.requestTime,
+        domainLookupStart: calculateOffset(timingPayload.dnsStart),
```

Comment:
Exposed timings need to be `timestamp` that denotes the event in RT API - https://w3c.github.io/resource-timing/#sec-performanceresourcetiming

I guess this is the closest we can go as far to keeping in Sync. Previous code was in millisecond which was not matching with RT spec.